execution-frameworks/Invoke-AtomicRedTeam/Invoke-AtomicRedTeam.ps1

```diff
@@ -0,0 +1,162 @@
+<#
+.SYNOPSIS
+This script will iterate over the Atomic Red Team yaml files, create objects for each test.
+The aim is to allow defenders to excercise MITRE ATT&CK Techniques to test defenses.
+
+
+Function: Invoke-AtomicRedTeam
+Author: Casey Smith @subTee
+License:  http://opensource.org/licenses/MIT
+Required Dependencies: powershell-yaml , Install-Module powershell-yaml #https://github.com/cloudbase/powershell-yaml
```

Comment:
I would directly call `Import-Module powershell-yaml` in the `Invoke-AtomicRedTeam.ps1` script.

This would reduce overall friction to the script.

Reply:
Ooops. What I meant was:

`Install-Module -Name powershell-yaml`

Reply:
I'd rather keep it out of Atomic and treat it as an external dependency